internal/i18n/i18n.go

```diff
@@ -139,6 +139,10 @@ func (t *Translator) Localize(id string, args map[string]interface{}, langs ...s
 	return localize(t.localizer(langs...), id, args)
 }
 
+func (t *Translator) LocalizeWithoutArgs(id string, langs ...string) string {
+	return localize(t.localizer(langs...), id, map[string]interface{}{})
+}
+
```

Comment:
not necessary if i18n keys are passed to the html

Reply:
kept it because its still checked for empty in the renderer..